pages/api/faucet/mumbai.ts

```diff
@@ -0,0 +1,81 @@
+import { ChainId } from "@thirdweb-dev/react";
+import { EVM_RPC_URL_MAP } from "constants/rpc";
+import { ethers } from "ethers";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method !== "POST") {
+    return res.status(400).json({ message: "Please use POST request" });
+  }
+
+  const { address } = JSON.parse(req.body);
+
+  const wallet = new ethers.Wallet(
+    process.env.PRIVATE_KEY as string,
+    new ethers.providers.JsonRpcProvider(EVM_RPC_URL_MAP[ChainId.Mumbai]),
+  );
```

Comment:
it is much more likely that we would set up a faucet as a smart contract than as a wallet